modules/ROOT/pages/administration/databases.adoc

```diff
@@ -707,10 +707,10 @@ Controls how the system handles existing data on disk when creating the database
 Currently this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
 
 | `existingDataSeedInstance`
-| instance ID of the cluster node
+| server ID of the cluster node
```

Comment:
We're trying to move away from using _node_ in this context. We don't use node in the operations manual so we should be consistent with that. I'd say something like: _ID of the cluster server_ or _server ID of the cluster instance_.
There are two more _node_s in this chapter that also need changing. I'll add a suggestion.

Reply:
I can't leave an inline comment, but this is on line 179:
`true` for the database node that accepts writes (this node is the leader for this database in a cluster or this is a standalone instance).
I suggest changing this to:
`true` for the database instance that accepts writes (this instance is the leader for this database in a cluster or this is a standalone instance).

Reply:
Sure thing! I've made the changes and pushed up. Feel free to hit merge